go-controller/pkg/networkmanager/nad_controller.go

```diff
@@ -273,6 +273,11 @@ func (c *nadController) syncNAD(key string, nad *nettypes.NetworkAttachmentDefin
 	if nad != nil {
 		nadNetwork, err = util.ParseNADInfo(nad)
 		if err != nil {
+			// in case the type for the NAD is not ovn-k we should not record the error event
+			if err.Error() == util.ErrorAttachDefNotOvnManaged.Error() {
```

Comment:
I knew this looked too familiar somewhere..

@martinkennelly ptal https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4846/files#diff-0b74e004432877a3ca7b23367059bb077ffa13c01d26f6e3a0f37d09acc67403R164
I thought we closed this PR because you saw the new syncNAD changes happen and didn't see this happening anymore?

@SchSeba : https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4846#issuecomment-2551576818
so is my understanding correct that error event here is causing noise for you? @martinkennelly maybe we should have merged your PR haha :)

if we still need this can we do `if err != nil && !errors.Is(err, config.ErrorAttachDefNotOvnManaged) {` ? its more cleaner...

Reply:
I think it will not be the same here @tssurya.

if I do `if err != nil && !errors.Is(err, config.ErrorAttachDefNotOvnManaged)` and the error is `config.ErrorAttachDefNotOvnManaged` it will add it to the list `nadNetworkName = nadNetwork.GetNetworkName()`

Reply:
/assign @martinkennelly